### PR TITLE
fix(web): breadcrumb navigations on list pages

### PR DESCRIPTION
**Pull request by m8vago**
Fix: Breadcrumb navigation on list pages while creating a new item.

### CODE REVIEW

File: web/crux-ui/Dockerfile [hunk 1]
```diff
@@ -8,30 +8,31 @@ COPY ./package.json ./package-lock.json ./.npm ./
...
 EXPOSE 3000
+ENV HOSTNAME "0.0.0.0"
 ENV PORT 3000
 LABEL org.opencontainers.image.source="https://github.com/dyrector-io/dyrectorio/web/crux-ui"
 
 HEALTHCHECK --interval=30s --timeout=5s \
     CMD wget -nv -t1 --spider http://localhost:3000/ || exit 1
 
-ENTRYPOINT [ "npm", "run", "start:prod"]
+CMD [ "node", "server.js" ]
```
Comment (nandor-magyar) [inline web/crux-ui/Dockerfile:38]: Using the ENTRYPOINT directive is preferable over CMD, though technically there is no much difference how it is executed.

**Review by nandor-magyar — CHANGES_REQUESTED** [review]

File: web/crux-ui/Dockerfile (hunk 1 above)
Comment (nandor-magyar) [reply web/crux-ui/Dockerfile:38]: We had a discussion that we need the entrypoint from the original image.
```sh
#!/bin/sh
set -e

# Run command with node if the first argument contains a "-" or is not a system command. The last
# part inside the "{}" is a workaround for the following bug in ash/dash:
# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=874264
if [ "${1#-}" != "${1}" ] || [ -z "$(command -v "${1}")" ] || { [ -f "${1}" ] && ! [ -x "${1}" ]; }; then
  set -- node "$@"
fi

exec "$@"
```

This is the content, so maybe just providing `server.js` would be enough then, to make it more apparent that there is also an entrypoint defined and it is in-use.

**Review by nandor-magyar — APPROVED** [review]

**Review by polaroi8d — APPROVED** [review]
Didn't tested in local, code looks okey for me.

File: web/crux-ui/e2e/utils/node-helper.ts [hunk 2]
```diff
@@ -8,12 +8,18 @@ export const installDagent = async (page: Page) => {
   await page.goto(TEAM_ROUTES.node.list())
   await page.waitForSelector('h2:text-is("Nodes")')
 
-  await page.locator('button:has-text("Add")').click()
+  await page.waitForTimeout(1000)
+
+  page.addListener('framenavigated', frame => console.info('PW navigated', frame.url()))
```
Comment (robot9706) [inline web/crux-ui/e2e/utils/node-helper.ts:13]: `console.info` :)

**Review by robot9706 — APPROVED** [review]

**Pull request merged**